htdocs/web_portal/views/site/edit_api_auth.php

```diff
@@ -67,6 +92,16 @@
                 <label class="input_label" for="ALLOW_WRITE">Allow API write</label>
             </div>
         </div>
-        <input type="submit" value="Edit credential" class="input_button">
+        <?php } ?>
+
+        <?php if ($params['isRenewalRequest']) {
+            echo '<br><p> Are you sure you want to continue? </P>';
+            echo '<input class="input_input_hidden" type="hidden" name="isRenewalRequest" value=true />';
```

Comment:
I think this can be displayed in both the `edit` or `renew` case

```suggestion
        <?php } ?>

        echo '<br><p> Are you sure you want to continue? </P>';
        <?php if ($params['isRenewalRequest']) {
            echo '<input class="input_input_hidden" type="hidden" name="isRenewalRequest" value=true />';
```

Reply:
Sure. As part of improving the edit functionality updated the codebase.